packages/frontend/src/components/TokenSelect.tsx

```diff
@@ -0,0 +1,88 @@
+import AppContext from './appContext';
+import { useContext, useEffect, useState } from 'react';
+import Axios from 'axios';
+import { useEthers } from '@usedapp/core';
+import { globalState } from '.';
+import { useAppContext } from './appContext';
+
+export default function TokenSelect() {
+  const [tokens, setTokens] = useState<any[]>([]);
+  const [selected, setSelected] = useState<any>('Select a token');
+  const [search, setSearch] = useState<any>('');
+  const { chainId, library } = useEthers();
+  const { setToken } = useAppContext();
+  // const signer = library!.getSigner();
+  const testWETH = {
+    address: '0xf298e212e92Dc2f0859309B5C5E9fDD3E29a9737',
+    name: 'TestWETH',
+    symbol: 'TWETH',
+    decimals: 18,
+    chainId: 4,
+    logoURI:
+      'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2/logo.png',
```

Comment:
- put the address in config file
- use the address to get name, symbol, decimals